ext/web/06_streams.js

```diff
@@ -633,6 +634,41 @@
     return stream[_disturbed];
   }
 
+  const DEFAULT_CHUNK_SIZE = 16_640;
```

Comment:
Have you experimented with greater size of this buffer?

Reply:
This change in unrelated to the optimization. It's just moving existing code from `ext/net` to `ext/web/06_streams.js`.  
Though yes, we should try increasing the chunk size to smth like 64KB, once #14265 lands to actually benefit from >16kb sizes.

Reply:
updated! and holy moly we are faster than Node now (see updated benchmarks above)